src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -2295,7 +2296,7 @@ public override IReadOnlyList<SyntaxNode> GetParameters(SyntaxNode declaration)
                 ? list.Parameters
                 : declaration is SimpleLambdaExpressionSyntax simpleLambda
                     ? new[] { simpleLambda.Parameter }
-                    : SpecializedCollections.EmptyReadOnlyList<SyntaxNode>();
+                    : SpecializedCollections.EmptyReadOnlyList<global::Microsoft.CodeAnalysis.SyntaxNode>();
```

Comment:
Not a problem, but why qualify the alias?

Reply:
looks like some refactoring did this.  i'm going to fixup.

Reply:
fixed.